src/pudl/package_data/settings/etl_fast.yml

```diff
@@ -113,4 +113,3 @@ datasets:
     # so if you're loading CEMS data for a particular year, you should
     # also load the EIA 860 data for that year if possible
     states: [ID, ME]
-    years: [2019, 2020, 2021]
```

Comment:
Was this an intentional deletion? It's making my `etl_fast` run take as long as it did to review the rest of this PR 😅

Reply:
Not intentional! My bad, should be back to normal now!

Reply:
Were the years added back?